### DISCUSSION TITLE
VSP Variable speed pump control without Aqualink support?

### DISCUSSION THREAD

**Discussion by weiyin** [aqualinkd/AqualinkD#304] [General]
I have an old Aqualink RS board from 2008 (I believe rev MM), that uses a PDA for control. I recently installed a VGreen EVO motor, which I currently operate standalone based on its internal schedule, but it can speak the Jandy ePump protocol.

Can AqualinkD control the pump speed if the Aqualink board is too old to support it?

**Comment by sfeakes** [comment]
No if can’t.  There can only be one master on a RS485 bus, and that’s the control panel. The master device has to control all other RS485 devices.  AqualinkD can read the pump / tell you what it’s doing, but it controls it through the panel.

**Comment by weiyin** [comment]
Ahh, okay makes sense. I guess a board upgrade is in my future. But I might try hacking something together with the CenturyVSPump project.
